### DISCUSSION TITLE
Commercial use

### DISCUSSION THREAD

**Discussion by PeterVD-Lansweeper** [vpnhood/VpnHood#64] [Q&A] [answered]
Hi,

Is it possible to use the library for a commercial product, without having to open source the proprietary code?
Is there some kind of commercial license?

Peter

**Comment by trudyhood** [comment, marked as answer]
Dear Peter,
Yes, LGPL allows you that; just include our license in your product.
Also, soon, we will launch an Access Server that lets you create your server farms and manage your users. It is optional, and you can also develop your access server with the REST interface offered by VpnHoodServer.

If you are interested, send an email to support@vpnhood.com and request to notify you when the beta is available. We are with you to help you run your business.

Feel free to submit any feature request that you think would help your business grows faster.
